src/main/java/jenkins/advancedqueue/jobinclusion/strategy/FolderBasedJobInclusionStrategy.java

```diff
@@ -73,8 +101,43 @@ public String getFolderName() {
 		return folderName;
 	}
 
+	public boolean isUseJobFilter() {
+		return useJobFilter;
+	}
+
+	public String getJobPattern() {
+		return jobPattern;
+	}
+
+	private Pattern getCompiledPattern() {
+		if (compiledPattern == null)
+			compiledPattern = Pattern.compile(jobPattern);
```

Comment:
Handle PatternSyntaxException somehow? Or at least add it to `throws`. If you follow the nullable field advice, it will need an additional check here as well